torch/onnx/__init__.py

```diff
@@ -224,13 +232,38 @@ def _run_symbolic_function(g, n, inputs):
             op_name = n.kind()[:-1]
         else:
             op_name = n.kind()
-        if not hasattr(torch.onnx.symbolic, op_name):
-            warnings.warn("ONNX export failed on {} because torch.onnx.symbolic.{} does not exist"
-                          .format(op_name, op_name))
-            return None
-        fn = getattr(torch.onnx.symbolic, op_name)
-        attrs = {k: n[k] for k in n.attributeNames()}
-        return fn(g, *inputs, **attrs)
+        if globals()['aten']:
+            attrs = {}
+            def type_suffix(var):
```

Comment:
In a situation like this, it might be better to call the underlying API, rather than construct the type suffixes, only for the helper function to immediately deconstruct the type suffix to reverse engineer the information you passed it. I'll also leave this up to you.

Reply:
Also, let us consider moving this to a helper function? :)

Reply:
Yep, let's move it to a helper function.